### PR TITLE
Cliu/homepage

### PR DESCRIPTION
**Pull request by changyang-liu**
Add a table of all galleries to the homepage, with pagination that retrieves galleries from the appropriate page as required. This PR will also include and utilize the antd package, which can be used in the future in other components.

I also added redux to the package.json as I realized that it was not included, even though it is required for react-redux.

### CODE REVIEW

**Review by DumboOctopus — CHANGES_REQUESTED** [review]
This some good code :ok_hand: v clean

Only two comments:
1. It looks like you started the branch off of lisha's incomplete select images component branch. I'm not too sure how this will impact when lisha does merge her code. But I think one thing we could do is replace some of lisha's files with the versions form the current main branch and then commiting thing. 
2. I think you will also need to modify one thing with the serializers in  `galleries/serailizers.py`. You will need to tell the GallerySerializer to serializer your new fields as well. Otherwise, the endpoint will never return a name or description, even if the gallery has one.

File: react/src/components/CreateUpdateGallery/index.js [hunk 1]
```diff
@@ -1,11 +1,242 @@
-import React from 'react'
+import React, { useState, useEffect, useReducer } from 'react'
+import axios from 'axios'
```
[the remark this answers is not in the document]
Comment (changyang-liu) [reply react/src/components/CreateUpdateGallery/index.js:2]: Oh right yeah I forgot about the serializer. I'll add that.

As for CreateUpdateGallery, is it already on main? If so I'll merge it and use the newest changes.

[the remark this answers is not in the document]
Comment (DumboOctopus) [reply react/src/components/CreateUpdateGallery/index.js:2]: On the main branch, I think that file has like nothing inside of it

[the remark this answers is not in the document]
Comment (changyang-liu) [reply react/src/components/CreateUpdateGallery/index.js:2]: Then I think we can just merge it later

[the remark this answers is not in the document]
Comment (DumboOctopus) [reply react/src/components/CreateUpdateGallery/index.js:2]: ok sounds good

**Review by DumboOctopus — APPROVED** [review]
Ok looks good! Feel free to click the merge button

**Pull request merged**